www-src/app/replicator/change/change_tag_handler.coffee

```diff
@@ -1,15 +1,25 @@
+AndroidCalendarHandler = require "../../lib/android_calendar_handler"
+CozyToAndroidCalendar = require "../transformer/cozy_to_android_calendar"
 log = require('../../lib/persistent_log')
     prefix: "ChangeTagHandler"
     date: true
 
 module.exports = class ChangeTagHandler
 
-    change: (doc) ->
-        log.info "change"
+    constructor: (@calendarSync) ->
+        @androidCalendarHandler = new AndroidCalendarHandler()
+        @cozyToAndroidCalendar = new CozyToAndroidCalendar()
 
-        console.log doc
+    dispatch: (cozyTag) ->
+        log.info "dispatch"
 
-    delete: (doc) ->
-        log.info "delete"
-
-        console.log doc
+        @androidCalendarHandler.getByName cozyTag.name, \
+                (err, androidCalendar) =>
```

Comment:
Actually, letting it may be more defensive to let only the AndroidCalendarHandler manage (delete) calendars, to keep the only strong criteria "delete a android calendar if it doesn't contains event anymore" as the only criteria to delete a calendar. Tags may be not that clean in the Cozy DB, and names duplicates may occurs (and then deletion of tag with name still used as calendar may occur ...)